pycfmodel/model/resources/properties/statement_condition.py

```diff
@@ -0,0 +1,328 @@
+    ForAnyValueNumericLessThanEqualsIfExists: Optional[Dict[str, ResolvableIntOrList]]
+    ForAnyValueNumericGreaterThanIfExists: Optional[Dict[str, ResolvableIntOrList]]
+    ForAnyValueNumericGreaterThanEqualsIfExists: Optional[Dict[str, ResolvableIntOrList]]
+
+    StringEquals: Optional[Dict[str, ResolvableStrOrList]]
+    StringNotEquals: Optional[Dict[str, ResolvableStrOrList]]
+    StringEqualsIgnoreCase: Optional[Dict[str, ResolvableStrOrList]]
+    StringNotEqualsIgnoreCase: Optional[Dict[str, ResolvableStrOrList]]
+    StringLike: Optional[Dict[str, ResolvableStrOrList]]
+    StringNotLike: Optional[Dict[str, ResolvableStrOrList]]
+    StringEqualsIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    StringNotEqualsIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    StringEqualsIgnoreCaseIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    StringNotEqualsIgnoreCaseIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    StringLikeIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    StringNotLikeIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringEquals: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringNotEquals: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringEqualsIgnoreCase: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringNotEqualsIgnoreCase: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringLike: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringNotLike: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringEqualsIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringNotEqualsIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringEqualsIgnoreCaseIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringNotEqualsIgnoreCaseIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringLikeIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAllValuesStringNotLikeIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringEquals: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringNotEquals: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringEqualsIgnoreCase: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringNotEqualsIgnoreCase: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringLike: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringNotLike: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringEqualsIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringNotEqualsIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringEqualsIgnoreCaseIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringNotEqualsIgnoreCaseIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringLikeIfExists: Optional[Dict[str, ResolvableStrOrList]]
+    ForAnyValueStringNotLikeIfExists: Optional[Dict[str, ResolvableStrOrList]]
```

Comment:
Any way we could autogenerate this from aws schema data (or, at very least check that we haven't forgotten any?)

Reply:
Resource specification does not include properties specification :(
All this combinations are based on this AWS docs page: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition_operators.html